apps/Gruntfile.js

```diff
@@ -606,6 +606,8 @@ describe('entry tests', () => {
       './src/sites/hourofcode.com/pages/public/index.js',
     'hourofcode.com/views/theme_common_head_after':
       './src/sites/hourofcode.com/pages/views/theme_common_head_after.js',
+    'hourofcode.com/views/hoc_events_map_replacement':
+      './src/sites/hourofcode.com/pages/views/hoc_events_map_replacement.js',
```

Comment:
At some point we might want to generalise this to be for any kind of embedded map?

Reply:
Maybe but there is a lot of custom logic in the JS file so generalizing the logic will be it's own task.

I definitely want to remove the `_replacement` suffix though once it's no longer a "replacement".